test/modules/GenericMqttTester/docker/amd64/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
+
+ARG EXE_DIR=.
+
+WORKDIR /app
+
+COPY $EXE_DIR/generic-mqtt-tester ./
+
+RUN apt-get update && apt-get install -y libssl-dev
+
+# Add an unprivileged user account for running the module
+RUN useradd -ms /bin/bash moduleuser
+USER moduleuser
+
+CMD echo "$(date --utc +"[%Y-%m-%d %H:%M:%S %:z]"): Starting Module" && \
+/app/generic-mqtt-tester
```

Comment:
Adding simple dockerfile for now, will change and add others when I make a PR for the build pipeline.